sync/_databricks.py

```diff
@@ -1481,6 +1481,7 @@ def _poll_for_eventlog_from_dbfs(
                 f"No or incomplete event log data detected - attempting again in {poll_duration_seconds} seconds"
             )
             sleep(poll_duration_seconds)
+            eventlog_response = None
```

Comment:
the s3 fetcher above this has the exact same issue btw

Reply:
Updated!